packages/typespec-test/test/openai_modular/generated/typespec-ts/review/openai_modular.api.md

```diff
@@ -129,12 +129,6 @@ export interface AzureChatExtensionDataSourceResponseCitation {
     url?: string;
 }
 
```

Comment:
Is this expected? it seems relevant to this change: https://github.com/Azure/autorest.typescript/pull/2556#discussion_r1682897313.